00_introduction/21_architecture_root_server.txt

```diff
@@ -204,11 +196,11 @@ The rsyslog server is receiving the logs from the nodes and insert them into a
 PostgreSQL database. On SLES, the +rsyslog-pgsql+ package is not part of the
 distribution, it can be downloaded alongside Rudder packages.
 
-Oracle Java JRE or OpenJDK 7 JRE::
```

Comment:
Please add the hard constraint of Postgres >= 8.4 and that much better performance are obtain with Postgres 9.1 and up.